extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/ArtnetBridgeHandler.java

```diff
@@ -0,0 +1,103 @@
+import org.eclipse.smarthome.binding.dmx.internal.multiverse.Universe;
+import org.eclipse.smarthome.config.core.Configuration;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ArtnetBridgeHandler} is responsible for handling the communication
+ * with ArtNet devices
+ *
+ * @author Jan N. Klug - Initial contribution
+ */
+public class ArtnetBridgeHandler extends DmxOverEthernetHandler {
+    public final static Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_ARTNET_BRIDGE);
+    public final static int MIN_UNIVERSE_ID = 0;
+    public final static int MAX_UNIVERSE_ID = 32767;
+
+    private final Logger logger = LoggerFactory.getLogger(ArtnetBridgeHandler.class);
+
+    public ArtnetBridgeHandler(Bridge artnetBridge) {
+        super(artnetBridge);
+    }
+
+    @Override
+    protected void updateConfiguration() {
+        Configuration configuration = getConfig();
+
+        int universeId = MIN_UNIVERSE_ID;
+        if (configuration.get(CONFIG_UNIVERSE) != null) {
+            universeId = Util.coerceToRange(((BigDecimal) configuration.get(CONFIG_UNIVERSE)).intValue(),
+                    MIN_UNIVERSE_ID, MAX_UNIVERSE_ID, logger, "universeId");
+        }
+
+        if (universe == null) {
+            universe = new Universe(universeId);
+        } else if (universe.getUniverseId() != universeId) {
+            universe.rename(universeId);
+        }
+
+        packetTemplate.setUniverse(universe.getUniverseId());
+
+        receiverNodes.clear();
+        if (configuration.get(CONFIG_ADDRESS) == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                    "Could not initialize sender (address not set) in ArtNet Bridge " + this.thing.getUID());
+            uninstallScheduler();
+            logger.error("remote address not set for {}", this.thing.getUID());
+            return;
+        } else {
+            receiverNodes = IpNode.fromString((String) configuration.get(CONFIG_ADDRESS), ArtnetNode.DEFAULT_PORT);
+            logger.debug("using unicast mode to {} for {}", receiverNodes.toString(), this.thing.getUID());
+        }
+
+        if (configuration.get(CONFIG_LOCAL_ADDRESS) != null) {
+            senderNode = new IpNode((String) configuration.get(CONFIG_LOCAL_ADDRESS));
+        }
+        logger.debug("originating address is {} for {}", senderNode, this.thing.getUID());
+
+        if (configuration.get(CONFIG_REFRESH_MODE) != null) {
+            refreshAlways = (((String) configuration.get(CONFIG_REFRESH_MODE)).equals("always"));
+        }
+        logger.debug("refresh mode set to always: {}", refreshAlways);
+
+        super.updateConfiguration();
+
+        updateStatus(ThingStatus.ONLINE);
```

Comment:
This is UDP , so connectionless. And the Bridge is not responding to the data send, so the best guess is to set the Bridge to online if it is possible to open a port to send data. That is what I'm doing.